docs/changelog.rst

```diff
@@ -1,6 +1,17 @@
 Changelog
 ~~~~~~~~~
 
+0.5.0 (October 2022)
```

Comment:
Should bump the version in `_metadata.py`.

Reply:
Also change to November :-)

Reply:
Thank you, forgot that.